deployment/Triton/README.md

```diff
@@ -0,0 +1,277 @@
+#ENTRYPOINT [ "tritonserver", "--model-repository=/models", "--log-verbose=1"]
+```
+Note: The Triton service expects a certain directory structure discussed in [Model Config File](#model-config-file) to load the model definitions.
+
+2. Next, the container with the Triton Service is run as a service (in background or separate terminal for demo).  In this example, the ports used by the Triton Service are set to `8090` for client communications.
+```Dockerfile:
+demo_app_image_name="monai_triton:demo"
+docker run --shm-size=128G --rm -p 127.0.0.1:8090:8000 -p 127.0.0.1:8091:8001 -p 127.0.0.1:8092:8002 ${demo_app_image_name}
+```
+3. See [Model Config File](#model-config-file) to see the expected file structure for Triton.
+- Modify the models/monai_prostrate/1/model.py file to satisfy any model configuration requirements while keeping the required components in the model definition. See the * [Usage](#usage) section for background. 
+- In the models/monai_prostrate/1/config.pbtxt file configure the number of GPUs and which ones are used.
+e.g. Using two available GPUs and two parallel versions of the model per GPU
+```
+instance_group [
+  { 
+    kind: KIND_GPU
+    count: 2
+    gpus: [ 0, 1 ]
+  }
+```
+e.g. Using three of four available GPUs and four parallel versions of the model per GPU
+```
+instance_group [
+  { 
+    kind: KIND_GPU
+    count: 4
+    gpus: [ 0, 1, 3 ]
+  }
+```
+Also, other configurations like dynamic batching and corresponding sizes can be configured. See the [Triton Service Documentation model configurations](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md) documentation for more information.
+ 
+- Finally, be sure to include you Tensors or Torchscript definition *.ts files in the directory structure. In this example,a prostate segmentation model based in PyTorch is used. 
+```
+prostate_model.ts
+```
+The Dockerfile will copy the model definition structure into the  Triton container Service.  So the container should be rebuilt after any modifications to the GPU or model configurations for the example.
+
+4. A Python [client](./client/client.py) program configures the model and makes an http request to Triton as a Service. Note: Triton supports other interfaces like gRPC.
+The client reads an input image converted from Nifti to a byte array for classification. 
+
+- In this example, a model trained to detect COVID-19 is provide an image with COVID or without.    
+```python:
+filename = 'client/test_data/volume-covid19-A-0000.nii.gz'
+```
+- The client calls the Triton Service using the external port configured previously.
+```python:
+with httpclient.InferenceServerClient("localhost:8090") as client:
+```
+- The Triton inference request is returned in the response 
+```python:
+response = client.infer(model_name,
+    inputs,
+    request_id=str(uuid4().hex),
+    outputs=outputs,)
+
+result = response.get_response()
+```
+-------
+## Usage
```

Comment:
May I ask a question abou this `Usage` part: Is it specific for this example or exists in some Triton tutorial? If the latter, I would suggest to put a link here to simplify the README doc.

Thanks.

Reply:
The latter, so I've referenced the sections that are essentially taken from the original Triton Python backend repo here:
## Usage
[See Triton Inference Server/python_backend documentation](https://github.com/triton-inference-server/python_backend#usage)
## Model Config
[See Triton Inference Server/python_backend documentation](https://github.com/triton-inference-server/python_backend#model-config-file)
## Error Handling
[See Triton Inference Server/python_backend documentation](https://github.com/triton-inference-server/python_backend#error-handling)